src/ExpressiveInstaller/Resources/config/database-doctrine.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Doctrine\ORM\EntityManager;
+use Kocal\Expressive\Database\Doctrine\EntityManagerFactory;
+
+return [
+    'dependencies' => [
+        'factories' => [
+            // Use EntityManagerFactory for using Doctrine EntityManager:
+            EntityManager::class => EntityManagerFactory::class
+        ]
+    ],
+
+    'doctrine' => [
+        // DBAL configuration:
+        // See http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html
+        'driver' => 'pdo_mysql',
```

Comment:
Yea you are right. 
I thought it was better to use pdo_mysql because I think MySQL/MariaDB is most used than SQLite, but in this case it's better to use pdo_sqlite.